src/stcal/jump/twopoint_difference.py

```diff
@@ -165,8 +165,6 @@ def find_crs(dataa, group_dq, read_noise, rejection_thresh,
 
         log_str = 'From highest outlier, two-point found {} pixels with at least one CR from {} groups.'
         log.info(log_str.format(len(row4cr), 'five or more'))
-        log.info(log_str.format(len(row3cr), 'four'))
```

Comment:
Any particular reason why you're removing this logging info? Is it no longer relevant?

Reply:
With multiprocessing this generates a lot of output that > 99% of the time doesn't show any jumps in the 2 and 3 group semi-ramps. It annoyed me. So, I thought it would annoy most other people too.